.ci/Jenkinsfile

```diff
@@ -300,3 +304,16 @@ def generateFunctionalTestStep(Map params = [:]){
     }
   }
 }
+
+/**
+ Notify the GitHub check of the parent stream
+**/
+def githubCheckNotify(String status) {
+  if (params.GITHUB_CHECK_NAME?.trim() && params.GITHUB_CHECK_REPO?.trim() && params.GITHUB_CHECK_SHA1?.trim()) {
+    githubNotify context: "${params.GITHUB_CHECK_NAME}",
+                 description: "${params.GITHUB_CHECK_NAME} ${status.toLowerCase()}",
+                 status: "${status}",
+                 targetUrl: "${env.RUN_DISPLAY_URL}",
+                 sha: params.GITHUB_CHECK_SHA1, account: 'elastic', repo: params.GITHUB_CHECK_REPO, credentialsId: env.JOB_GIT_CREDENTIALS
```

Comment:
`JOB_GIT_CREDENTIALS` is not defined in the environment section [L7](https://github.com/elastic/e2e-testing/pull/315/files#diff-8db37e8fcea0f1a8f2f39667e94ebcc4R7)

Reply:
Added, good catch!